### PR TITLE
chore(ci): converted travis workflow to github action

### PR DESCRIPTION
**Pull request by marcuslyons**
Ref [Level/community#99](https://github.com/Level/community/issues/99)

### CODE REVIEW

File: .github/workflows/sauce.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Sauce Labs
+on: push
+concurrency: sauce-labs
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```
Comment (vweevers) [inline .github/workflows/sauce.yml:9]:
```suggestion
        uses: actions/checkout@v2
```

The action we copied from was a bit outdated

File: .github/workflows/sauce.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Set up node
+        uses: actions/setup-node@v1
```
Comment (vweevers) [inline .github/workflows/sauce.yml:11]:
```suggestion
        uses: actions/setup-node@v2
```

Comment (marcuslyons) [reply .github/workflows/sauce.yml]: Changed in b660785

File: .github/workflows/sauce.yml (hunk 1 above)
Comment (marcuslyons) [reply .github/workflows/sauce.yml]: Changed in b660785

**Pull request merged**